skorch/callbacks/lr_scheduler.py

```diff
@@ -123,6 +115,15 @@ def initialize(self):
         self.policy_ = self._get_policy_cls()
         self.lr_scheduler_ = None
         self.batch_idx_ = 0
+        # TODO: Remove this warning on 0.10 release
+        if self.policy_ == TorchCyclicLR or self.policy_ == "TorchCyclicLR":
```

Comment:
Only need to warn if `step_every` is `'epoch'`.

```suggestion
        if self.policy_ == TorchCyclicLR or self.policy_ == "TorchCyclicLR" and self.step_every == 'epoch':
```

We should also have a test to make sure the warning message is raised.